test/system/update_patient_info_test.rb

```diff
@@ -82,7 +82,7 @@ class UpdatePatientInfoTest < ApplicationSystemTestCase
 
       it 'should update appointment date' do
         within :css, '#patient_dashboard' do
-          assert has_field? 'Appointment date', with: 5.days.from_now.strftime('%Y-%m-%d')
+          assert has_field? 'Appointment date', with: 5.days.from_now.strftime('%m/%d/%Y')
```

Comment:
What's up with the date format from before passing? Weren't we always m/d/y? 🤔

Reply:
Disregard this, I was trying to debug some spec failures and this local change snuck in. It was unnecessary.